### PR TITLE
MIR-724 Add integration tests for search behavior

### PR DESCRIPTION
**Pull request by KelKx**
added integration tests for searchfield, simple search and complex search

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 0c42e746bcad6c3a97e7301d320300c871d7746d! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1328-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 0915abf9b41fda7081740887a39f2404740fe1b4! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1335-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 05ec999273688776d954d40fb30f945ed1ecfc77! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1370-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 167bbc719d049297964494799aaf3df004d8466d! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1382-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 167bbc719d049297964494799aaf3df004d8466d! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1382-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 167bbc719d049297964494799aaf3df004d8466d! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1382-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 566884bb7e0c8e5a4b1ed9c010b6b4ace8b66b66! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1384-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 0cbdd862a32c5c9a917d452d3a907c0b17208fc0! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1464-pull_request-237

**Comment by kkrebs** [comment]
there are dependency problems:

```
[WARNING] Used undeclared dependencies found:
[WARNING]    org.mycore:mycore-base:jar:2017.07-SNAPSHOT:test
[WARNING]    com.google.code.gson:gson:jar:2.8.2:test
[INFO] Add the following to your pom to correct the missing dependencies: 
[INFO] 
<dependency>
  <groupId>org.mycore</groupId>
  <artifactId>mycore-base</artifactId>
  <version>2017.07-SNAPSHOT</version>
  <scope>test</scope>
</dependency>
<dependency>
  <groupId>com.google.code.gson</groupId>
  <artifactId>gson</artifactId>
  <version>2.8.2</version>
  <scope>test</scope>
</dependency>

```

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with d4cb7d34bed8bb86c7461fba8c0f5ce737853c32! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1493-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with d601a77962088b068574e486923c456ba89b272c! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1495-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with a15654f5f036c7ac89781f84e86050e14fb93b57! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1497-pull_request-237

File: mir-it/src/test/java/org/mycore/mir/it/tests/MIRSimpleSearchITCase.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Before
+    public final void ini() throws IOException, InterruptedException {
+        MIRSearchTestDataLoader searchTestDataLoader = new MIRSearchTestDataLoader();
+        searchTestDataLoader.lazyLoadData(getDriver());
+
+    }
+
+    @Test
+    public void testForm() {
```
[the remark this answers is not in the document]
Comment (paschty) [reply mir-it/src/test/java/org/mycore/mir/it/tests/MIRSimpleSearchITCase.java]: It includes assert in forEach

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 53b19d1a6319044be7af4aeafb7e58038a5618c0! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1587-pull_request-237

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 6624629efe9c48ad140bb059c58ebfee8795c01f! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1607-pull_request-237

**Pull request merged**